### DISCUSSION TITLE
quarter day counter

### DISCUSSION THREAD

**Discussion by lazyfvx** [community/community#38552] [Other Feature Feedback, Questions, & Ideas]
Who can help make a code to counter days in each quarter(every 3 months from January).For example contract starts 01.01.2023 and ends 05.01.2023.Its in first and second quarter i need code to auto counter how much days of the contract in each quarter.You input contract duration and output is how much days in each quarter.

**Comment by david-engelmann** [comment]
If you are using python
```
import datetime
x = input("Enter first date in Y,m,d format:")
y = input("Enter second date in Y,m,d format:")
a = datetime.datetime.strptime(x,"%Y,%m,%d")
b = datetime.datetime.strptime(y,"%Y,%m,%d")
(b-a).days
```

**Reply by lazyfvx** [reply]
Thanks, but no output. 1quarter-days 2quarter-days 3quarter-days 4quarter- days.

**Reply by david-engelmann** [reply]
Set the first day of quarter 1 to x and the last of quarter 1 to y and (b-a).days is equal to 1 quarter days
